[rospy.client][INFO] 2021-11-26 09:10:05,947: init_node, name[/server_endpoint], pid[499]
[xmlrpc][INFO] 2021-11-26 09:10:05,947: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-11-26 09:10:05,947: Started XML-RPC server [http://wng:43357/]
[rospy.init][INFO] 2021-11-26 09:10:05,947: ROS Slave URI: [http://wng:43357/]
[rospy.impl.masterslave][INFO] 2021-11-26 09:10:05,947: _ready: http://wng:43357/
[rospy.registration][INFO] 2021-11-26 09:10:05,947: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-11-26 09:10:05,948: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-11-26 09:10:06,047: registered with master
[rospy.rosout][INFO] 2021-11-26 09:10:06,048: initializing /rosout core topic
[rospy.rosout][INFO] 2021-11-26 09:10:06,048: connected to core topic /rosout
[rospy.simtime][INFO] 2021-11-26 09:10:06,049: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-11-26 09:10:06,053: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-11-26 09:10:06,297: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-11-26 09:10:16,548: Connection from 192.168.1.146
[rosout][INFO] 2021-11-26 09:10:16,552: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-11-26 09:10:16,577: topic[/execution_check] adding connection to [http://wng:43357/], count 0
[rospy.internal][INFO] 2021-11-26 09:10:16,578: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-11-26 09:50:42,663: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-11-26 09:50:42,666: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-11-26 09:50:42,667: topic[/rosout] removing connection to /rosout
[rospy.internal][ERROR] 2021-11-26 09:50:42,666: Unable to initiate TCP/IP socket to wng:40727 (http://wng:43357/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-11-26 09:50:42,667: topic[/execution_check] removing connection to http://wng:43357/
[rospy.impl.masterslave][INFO] 2021-11-26 09:50:42,667: signal-2
[rospy.core][INFO] 2021-11-26 09:50:42,785: signal_shutdown [atexit]
